Healthy/Classes/Utilities/ValidatorProtocol/HealthyValidator .swift

```diff
@@ -0,0 +1,38 @@
+//
+//  HealthyValidator .swift
+//  Healthy
+//
+//
+
+import Foundation
+public protocol HealthyValidator {
```

Comment:
Great job making this extensible !!!  [LGTM](https://media.giphy.com/media/NytMLKyiaIh6VH9SPm/giphy.gif)